test/surveys/send/SendTests.cs

```diff
@@ -8,12 +7,12 @@
 using Newtonsoft.Json;
 using NUnit.Framework;
 
-namespace com.esendex.sdk.test.surveys
+namespace com.esendex.sdk.test.surveys.send
 {
     [TestFixture]
-    public class SurveysServiceTests
+    public class SendTests

```

Comment:
Ambiguous without the namespace. If doing a search through the solution while developing it may not be clear what these tests are testing.